src/FavoritesList.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+
+export default function FavoritesList() {
+  let favorites = localStorage.getItem("favorites");
+  let favoritesArray = JSON.parse(favorites);
+  let favoritesSet = new Set();
+  favoritesSet = new Set(favoritesArray);
+  let arrayFromSet = Array.from(favoritesSet);
+
+  return (
+    <>
+      <Link to={"/"}>Back to list</Link>
+      <ul>
+        {arrayFromSet.map((names) => {
```

Comment:
"names" ist nicht so gut gewählt als Name für eine Variable in diesem Fall, weil nameS suggestiert dass es plural (ergo ein Array oder so) ist. In wirklichkeit ist es aber nur ein einzelner String/Name.

"pokemonName" wäre besser

Reply:
Jo, das ist mir in dem Moment nicht eingefallen. Bin endlich mal dazu gekommen auszubessern